sfm_pc/views.py

```diff
@@ -264,38 +267,36 @@ def download_zip(request):
     return resp
 
 
-class DownloadData(FormView):
+class DownloadData(TemplateView):
     template_name = 'download.html'
-    form_class = DownloadForm
-    success_url = reverse_lazy('download')
 
-    def form_valid(self, form):
+    def get_context_data(self):
+        context = super().get_context_data()
 
-        download_type = form.cleaned_data['download_type']
-        division_id = form.cleaned_data['division_id']
-        confidences = form.cleaned_data['confidences']
+        download_url = self.get_presigned_url()
```

Comment:
Is this page cached in deployment?

Reply:
Actually, the cache is only for the zip archive that was served from a view. The download page itself wasn't cached.